### DISCUSSION TITLE
App Slowdown

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#450] [General]
I'm working on a rigged character to do storytime animation on YouTube. I'm using an iPad Air 4, and I notice some slowdown in the app as I work on the rig. I exported it to my 12.9 inch iPad Pro 4, and the same slowdown thing occurs. As a realist, I expected this to happen eventually, but could it be that I need to get a more powerful iPad, or is this something a future update could take care of. I prefer working on my iPad Air 4 because of its more portable size, and I don't currently have the money to get one of the high end iPad Pros. As for the rig itself, I figured I'd animate the hands and feet the same way Rayman's are, and add the arms and legs with hand drawn animation later.

**Comment by keiwando** [comment]
Without having the project file, there is no way for me to tell what exactly the cause of the slowdown is. If you can send it to contact@toonsquid.com (either directly or a download link where I can download a copy of it if it's too large), I'll be more than happy to take a closer look. I'm always interested in real use-cases that reach the current performance limits of the app in order to hopefully improve the performance even more in the future, if possible.

**Reply by DWrightAnimation** [reply]
For some reason, the slow down thing worked itself out and I managed to complete my project on my iPad Pro without any further lag. But since I have your attention, could you add a button to allow us to edit a previously edited out of pegs onion skin without having to reset it? Look at RoughAnimator and Callipeg's out of pegs functionality to see what I mean.
